app/views/visitors/index.html.erb

```diff
@@ -1 +1,3 @@
-<h3>Welcome</h3>
+<%= content_for :page_title_prefix, I18n.t('page_title.home') %>
```

Comment:
This is the fault of the rails-template for having an `erb` file where we currently have `haml` as our default for Rails apps, can we switch over to haml?

Reply:
I did wonder!!